manuscript/manuscript.Rmd

```diff
@@ -369,7 +376,7 @@ plot_sims(tip_sims_df) / ( plot_policy(tip_policy_df) + plot_reward(tip_sims_df)
 
 The TD3-trained agent proves far more successful in preventing chance transitions across the tipping point, consistently achieving a higher cumulative ecosystem service value across replicates than the steady-state strategy.
 
-Examining the replicate management trajectories and corresponding rewards [Fig 5] reveal that the RL agent incurs significantly higher costs in the initial phases of the simulation, dipping the cumulative reward into negative territory on average and well below the reward realized by the simple steady-state strategy.
+Examining the replicate management trajectories and corresponding rewards [Fig 5] reveal that the RL agent incurs significantly higher costs in the initial phases of the simulation, dipping the cumulative reward into negative territory on average and well below the reward realized by the simple steady-state strategy. <!-- ML: Nit picky; it looks like it almost goes negative but not quite on average in the plot -->
```

Comment:
good catch, I think didn't revisit this text after I retrained the algo.  Let's just reword this to "dipping well below the mean steady-state reward initially before exceeding it in the long run"  or something like that.